.github/CODEOWNERS

```
@@ -17,6 +17,7 @@ source/adapters/native_cpu          @oneapi-src/unified-runtime-native-cpu-write
 source/adapters/**/command_buffer.*  @oneapi-src/unified-runtime-command-buffer-write
 scripts/core/EXP-COMMAND-BUFFER.rst  @oneapi-src/unified-runtime-command-buffer-write
 scripts/core/exp-command-buffer.yml  @oneapi-src/unified-runtime-command-buffer-write
+test/conformance/exp_command_buffer**
```

guessing we should specify the review group too
```suggestion
test/conformance/exp_command_buffer** @oneapi-src/unified-runtime-command-buffer-write
```

---

Apologies that was a simple oversight, fixed now!